scene/animation/animation_mixer.cpp

```diff
@@ -1073,15 +1076,18 @@ void AnimationMixer::_blend_calc_total_weight() {
 		real_t weight = ai.playback_info.weight;
 		Vector<real_t> track_weights = ai.playback_info.track_weights;
 		Vector<int> processed_indices;
-		for (int i = 0; i < a->get_track_count(); i++) {
-			if (!a->track_is_enabled(i)) {
+		const Vector<Animation::Track *> tracks = a->get_tracks();
```

Comment:
```suggestion
		const Vector<Animation::Track *> tracks = a->get_tracks();
```

I remember we always saying that we should use `LocalVector` whenever possible. There are several other places in the animation code where `Vector` is used besides here, but as long as it doesn't use `has()`, `sort()`, etc., I think it can be migrated to `LocalVector`.

Reply:
Using `LocalVector` can result in many copies. Also, if it is changed, it will be necessary to make many changes in the already existing code. For example, with `int` in for loop, replace it with `uint32_t` so that there are no warnings from the compiler.
In order to remove the performance overhead, I used `ptr`.